src/components/swiper/index.js

```diff
@@ -80,10 +95,12 @@ const Swiper = ( props ) => {
 
 	useEffect( () => {
 		try {
-			const swiperContainer = document.getElementById( uuid );
+			const currentDocument = determineDocumentObject();
```

Comment:
This is fine to have but recommend we simplify the logic around `determineDocumentObject`.